ee/agent/flags/flag_controller.go

```diff
@@ -647,3 +647,10 @@ func (fc *FlagController) LocalDevelopmentPath() string {
 		WithDefaultString(fc.cmdLineOpts.LocalDevelopmentPath),
 	).get(nil)
 }
+
+func (fc *FlagController) SetDesktopRunnerServerURL(url string) error {
```

Comment:
Passing this through a flag feels a little weird to me. Might be right though...  Why go this way?

Reply:
It does feel weird. I just followed the pattern used to store other data in the sqlite db via the start up settings writer. Perhaps we should create a new table / store in the sqlite db ... `runtime_values` or something?

Reply:
I like `runtime_values`